pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -132,11 +132,36 @@ class Build {
     The test jobs all follow the same name naming pattern that is defined in the openjdk-tests repository.
     E.g. Test_openjdk11_hs_sanity.system_ppc64_aix
     */
-    def getTestJobParams(testType) {
-        def jobParams = [:]
+     def getSmokeTestJobParams() {
+        def jobParams = getCommonTestJobParams()
+        jobParams.put('LEVELS', "extend")
+        jobParams.put('GROUPS', "functional")
+        jobParams.put('TEST_JOB_NAME', "${env.JOB_NAME}_SmokeTests")
+        jobParams.put("VENDOR_TEST_REPOS", ((String)ADOPT_DEFAULTS_JSON['repository']['build_url']).minus(".git"))
+        jobParams.put("VENDOR_TEST_BRANCHES","${ADOPT_DEFAULTS_JSON['repository']['build_branch']}")
+        jobParams.put("VENDOR_TEST_DIRS", "${ADOPT_DEFAULTS_JSON['repository']['test_dirs']}")
```

Comment:
These may not always use `ADOPT_DEFAULTS_JSON`. If this is being executed from a user's repository and perspective, this will pull in Adopt's vendor details.

Could we have a check to see if the user's respective `DEFAULTS_JSON` values match Adopt's and, if they don't, the user's DEFAULTS_JSON parameters should be used instead? Let me know if you need further explanation of what I mean 🙂

Reply:
Thanks @M-Davies ! Updated with using `buildConfig.USE_ADOPT_SHELL_SCRIPTS`, which I think should handle the case of executing from a user's perspective.

Reply:
Yep looks good! Thanks